lighthouse-core/lib/proto-preprocessor.js

```diff
@@ -23,13 +23,22 @@ function processForProto(result) {
   const reportJson = JSON.parse(result);
 
   // Clean up the configSettings
+  // Note: This is not strictly required for conversion if protobuf parsing is set to
+  //       'ignore unknown fields' in the language of conversion.
   if (reportJson.configSettings) {
-    // make sure the 'output' field is an array
-    if (reportJson.configSettings.output) {
-      if (!Array.isArray(reportJson.configSettings.output)) {
-        reportJson.configSettings.output = [reportJson.configSettings.output];
-      }
-    }
+    // Filter out unwanted fields
```

Comment:
~~wait I thought these were the only ones we were keeping?~~ oh I see that is what's happening :)

maybe rename the variable so the comment isn't necessary (i.e. `onlyFieldsToKeep` or something) or flesh it out a bit so it's clear `// We'll discard all fields except these because LR doesn't need them` or w/e?